clap_generate/src/lib.rs

```diff
@@ -178,6 +178,9 @@ where
     G: Generator,
     S: Into<String>,
 {
+    #[cfg(debug_assertions)]
+    let _ = env_logger::builder().format_timestamp(None).try_init();
+
```

Comment:
Let's use a custom cfg here, not `debug_assertions`. Loggers are **not** to be enabled by libraries. Internal hack should be next to impossible to trigger accidentally.

Reply:
If we use custom feature here, then we don't really need `log` and `env_logger`. We can keep the old `debug` feature and just take the improvements to the macros I did here.

Reply:
Well, I don't see them as improvements. We can very well do the same fancy `[DEBUG]` format with old macros.